### DISCUSSION TITLE
Timer Activities Missing in v3

### DISCUSSION THREAD

**Discussion by sagargolu** [elsa-workflows/elsa-core#3862] [Q&A]
@sfmskywalker, Do we have any timer activities in v3 or any vision to introduce it? Because I am not seeing it.

**Comment by jdevillard** [comment]
Hello @sagargolu , 

it's seems that there is already some Time activities in V3 : 

![image](https://user-images.githubusercontent.com/3254429/228967168-146364bf-96b4-4b34-a8ed-ced69519e37f.png)


To use it, you need to use the Scheduling Module : 

```csharp
   // Use timers.
    elsa.UseScheduling();
```

You can take a look at the `Elsa.Samples.WorkflowServer` project

**Reply by sagargolu** [reply]
Thanks for the quick response. Do we have some samples to use that activity? Or If you can provide some of the basics of it how will it work?
